drivers/usb/dwc2/platform.c

```diff
@@ -468,6 +471,31 @@ static int dwc2_driver_probe(struct platform_device *dev)
 	if (retval)
 		return retval;
 
+	/* Temporary workaround for USB FS clock and PHY initialization on STM32H7. */
```

Comment:
Let's re-phrase  this... something like "Additionally initialize *** on STM32H7"